contrib/uriparser/example/main.cc

```diff
@@ -0,0 +1,116 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include "absl/flags/parse.h"
+#include "contrib/uriparser/sandboxed.h"
+#include "contrib/uriparser/utils/utils_uriparser.h"
+
+void Print(const char* name, absl::StatusOr<std::string> r) {
```

Comment:
`r` should be a const reference to avoid an extra copy: `const absl::StatusOr<std::string>&`

Reply:
Fixed.